requirements.txt

```diff
@@ -0,0 +1,4 @@
+colorlog==6.8.2
+homeassistant==2024.6.4
```

Comment:
Why is this hass pinned here?

Reply:
In the new devcontainer setup, HASS dependency is pinned here in requirements. These dependencies are for development only and are installed by the `setup` script. In the previous devcontainer setup I believe HASS was installed by default in the container image. The previous image also contained a script for selecting HASS version.

I chose 2024.6 because of #434

Reply:
Please drop numpy and unpin homeassistant. We want the latest regular version